frontend/src/stories/components/Users/UsersTable.stories.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import UsersTable from 'main/components/Users/UsersTable';
+import usersFixtures from 'fixtures/usersFixtures';
+
+export default {
+    title: 'components/users/UsersTable',
+    component: UsersTable
+};
+
+const Template = (args) => {
+    return (
+        <UsersTable {...args} />
+    )
+};
+
+export const Empty = Template.bind({});
+
+Empty.args = {
+    users: []
+};
+
+export const ThreeUsers = Template.bind({});
+
+let threeUsersCopy = JSON.parse(JSON.stringify(usersFixtures.threeUsers));
+threeUsersCopy[2]["hidden"] = true;
+
+ThreeUsers.args = {
+    users: threeUsersCopy
+};
+
+export const OneUsers = Template.bind({});
+
+OneUsers.args = {
+    users: usersFixtures.threeUsers.slice(0,1)
+}
+
+/*
+export const ThreeCommonsAdmin = Template.bind({});
+
+ThreeCommonsAdmin.args = {
+    commons: commonsPlusFixtures.threeCommonsPlus,
+    currentUser: currentUserFixtures.adminUser
+};
+
+export const OneCommonsAdmin = Template.bind({});
+
+OneCommonsAdmin.args = {
+    commons: commonsPlusFixtures.oneCommonsPlus,
+    currentUser: currentUserFixtures.adminUser
+}
+*/
```

Comment:
commented out code?

Reply:
Again this is leftover code copied from Commons story. I will remove this.

Reply:
https://github.com/ucsb-cs156-m23/proj-happycows-m23-10am-4/pull/86/commits/79239c3485bc4b8a3cd5c2754f9dbc573cc9e8d1